src/flow_node_instance_repository.ts

```diff
@@ -374,10 +386,15 @@ export class FlowNodeInstanceRepository implements IFlowNodeInstanceRepository,
                               previousFlowNodeInstanceId: string): Promise<Runtime.Types.FlowNodeInstance> {
 
     const createParams: any = {
```

Comment:
Isn't it `IFlowNodeInstanceAttributes`?

```suggestion
    const createParams: IFlowNodeInstanceAttributes = {
```